db/models.py

```diff
@@ -42,11 +45,75 @@
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
     cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=CinemaHall,
+        on_delete=models.CASCADE,
+        related_name="movie_sessions"
     )
     movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=Movie, on_delete=models.CASCADE,
+        related_name="movie_sessions"
     )
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        related_name="orders")
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"<Order: {self.created_at}>"
+
+
+class Ticket(models.Model):
+    order = models.ForeignKey(
+        "Order",
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    movie_session = models.ForeignKey(
+        "MovieSession",
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(
+                fields=["row", "seat", "movie_session"],
+                name="unique_ticket_constraint")
+        ]
+
+    def clean(self) -> None:
+        cinema_hall = self.movie_session.cinema_hall
+        if not (1 <= self.row <= cinema_hall.rows):
+            raise ValidationError(
+                {"row": f"row number must be "
+                    f"in available range: (1, rows): "
+                    f"(1, {cinema_hall.rows})"})
+        if not (1 <= self.seat <= cinema_hall.seats_in_row):
+            raise ValidationError(
+                {"seat": f"seat number must "
+                    f"be in available range: (1, seats_in_row): "
+                    f"(1, {cinema_hall.seats_in_row})"})
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+    def __str__(self) -> str:
+        return (f"<Ticket: {self.movie_session} "
+                f"(row: {self.row}, seat: {self.seat})>")
+
+
+class User(AbstractUser):
```

Comment:
The `User` model inherits from `AbstractUser` but currently does not add any additional fields or methods. If no customization is needed, consider whether extending `AbstractUser` is necessary, or if a simple `User` model suffices.